integration-tests/main.go

```diff
@@ -54,6 +54,10 @@ func (tr TestRun) runStep(step Step, verbose bool) {
 		tr.delegateTokens(action, verbose)
 	case UnbondTokensAction:
 		tr.unbondTokens(action, verbose)
+	case SlashAction:
+		tr.InvokeSlash(action, verbose)
```

Comment:
In general, only export a method if necessary. i.e. this should probably be lower case

Reply:
I went ahead and changed each integration test action and corresponding functions to be non-exported